content/docs/dvclive/api-reference/next_step.md

```diff
@@ -0,0 +1,52 @@
+# dvclive.next_step()
+
+Signals that the current step has ended. Check the
+[init parameters](/doc/dvclive/api-reference/init#parameters) for configuring
+the behavior.
```

Comment:
> Signals that the current step has ended.

What's a step? A step of what?

Reply:
> Check the init parameters...

This info should probably be in a quick `init()` mention at the beginning of the **Description**, I think (same as I mentioned for `dvclive.log`).